archinstall/lib/installer.py

```diff
@@ -51,6 +51,10 @@ def __init__(self, target, *, base_packages=__packages__[:3], kernels=['linux'])
 		self.base_packages = base_packages.split(' ') if type(base_packages) is str else base_packages
 		for kernel in kernels:
 			self.base_packages.append(kernel)
+		if hasUEFI():
+			self.base_packages.append("efibootmgr")
+		else:
+			self.base_packages.append("grub")
```

Comment:
I don't like this revert, it doesn't make sense to guess and strap in packages that may or may not be used. I fail to see how this would fix the problem as well seeing as `grub` will never be installed but we'll call `grub-install` later in the code. I think we should patch line 433 instead with:
```python
self.pacstrap('grub')
if hasUEFI():
    self.pacstrap('efibootmgr')`

Reply:
Changed in latest commit.